studio-docs/source/schema-checks.mdx

```diff
@@ -88,23 +122,48 @@ You override flagged changes on an operation-by-operation basis. For each operat
 * **Ignore the operation.** In this case, schema checks will completely ignore the operation when checking _all_ changes in any future execution.
   * This option is useful when you know an operation originates _only_ from clients or client versions that you don't actively support.
 
-### Rerunning a check
+### Rerunning checks
 
-You can also rerun checks from inside Studio. When you rerun a check, the latest check configuration is applied. The new check uses an updated time window for operations.  In addition, any changes made to excluded / included clients, checked variants, and any operations already marked safe / ignored are incorporated when rerunning.
+You can rerun checks from inside Studio:
+
+<img class="screenshot" src="./img/schema-checks/rerun-check.png" alt="Rerun check in Apollo Studio" />
+
+When you do, the new check uses the current [check configuration options](./check-configurations/), regardless of the configuration at the time of the original check. Similarly, the new check's time window is based on the current time, _not_ the time when the original check ran.
+
+The new check incorporates any changes made to excluded or included clients, checked variants, and any operations marked as safe or ignored.
 
 > **Note:** If you've [integrated schema checks with your GitHub PRs](#integrating-with-github), a rerun of the check also updates the status of the check in Github.
 
-<img class="screenshot" src="./img/schema-checks/rerun-check.png" alt="Rerun check in Apollo Studio" />
+## Federated composition checks
 
-### Checking a federated schema
+If you have a [federated graph](https://www.apollographql.com/docs/federation/), you run schema checks with the `rover subgraph check` command (instead of `rover graph check`):
 
-If your data graph uses [Apollo Federation](https://www.apollographql.com/docs/federation/), the schema checks process differs slightly. For details, see [Checking changes to a federated graph](https://www.apollographql.com/docs/federation/managed-federation/advanced-topics/#checking-changes-to-a-federated-graph).
+```bash
+rover subgraph check docs-example-graph@current --schema ./schema.graphql --name products
+```
+
+In addition to the arguments required by `graph check` , `subgraph check` requires the `--name` of the particular subgraph you're checking against.
+
+When you run `subgraph check`, Apollo Studio performs operation checks as usual, along with **composition checks**: Studio verifies that the changes you make to a subgraph schema are compatible with your _other_ subgraph schemas, enabling them to compose into a _supergraph_ schema ([learn more](https://www.apollographql.com/docs/federation/quickstart/#federation-concepts)).
```

Comment:
This is looking awesome! The one thing to call out here is that for a federated graph, we actually run composition checks _before_ operation checks. So if composition fails, then we just don't run operation checks. We should add this bit of information somewhere here :)